### ISSUE TITLE
backup:cleanup task is unable to chmod a blank string...go figure

### ISSUE THREAD

**Issue by grafikchaos** [augustash/capistrano-ash#44]
`chmod: missing operand after '755'`

real live error during a deployment:

``` shell
executing "find /path/to/app/staging/backups/magento_staging_dump-2013-06-24_18:17:37_-0500.sql.gz -type d -print0 | xargs -0 chmod 755"
```

**Comment by grafikchaos** [comment]
Looks to be related to the PR from #38

**Comment by grafikchaos** [comment]
fix is pretty simple:

in [ash/base around line 426](https://github.com/augustash/capistrano-ash/blob/develop/lib/ash/base.rb#L426) do the following:

``` ruby
set_perms_dirs("#{backup}", 755) if File.directory?(backup)
```

**Comment by grafikchaos** [comment]
Released new capistrano-ash version [1.3.4](http://rubygems.org/gems/capistrano-ash)

``` bash
$ gem install capistrano-ash
```
